src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -8,8 +8,6 @@ translate:
   - jp
 metaDescription: How the New Relic One pricing plan and billing calculations work.
 redirects:
-  - /docs/accounts/original-accounts-billing/product-based-pricing/original-product-based-pricing
```

Comment:
Not sure about removing this one. With this gone http://localhost:8000/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing results in a 404. Was that the goal?